ratatui/src/widgets/list/list.rs

```diff
@@ -296,8 +297,8 @@ impl<'a> List<'a> {
     /// let list = List::new(items).highlight_symbol(">>");
     /// ```
     #[must_use = "method moves the value of self and returns the modified value"]
-    pub const fn highlight_symbol(mut self, highlight_symbol: &'a str) -> Self {
```

Comment:
This is a (small) breaking change, which should be noted in the breaking changes doc and commit (the first PR comment will be used as the squashed commit message).
- breaks inference
- breaks const usage

Can you please search for similar changes in the docs and change log and copy the wording?

Reply:
You can check out older PR's that made breaking changes. 

For example take a look at these PR's
- https://github.com/ratatui/ratatui/pull/1418
- https://github.com/ratatui/ratatui/pull/1331

The first comment contains a general description of the change, followed by the breaking changes written as "BREAKING CHANGE: \<the change that is breaking\>"

As for the title, it should be something like "refactor!: ..." with an exclamation point to indicate that it was a breaking change.

At last you will also want to update the breaking changes doc with the relevant information. See these commits for an example: 
- https://github.com/ratatui/ratatui/pull/1331/commits/ea10843985eb873dae113d550ff63b16e1f2ebf4
- https://github.com/ratatui/ratatui/pull/1418/commits/62c7fb9dca12e11a5510501ba3422f05d87ac398

In simpler terms your initial PR comment is used as a commit message, and the PR title is used as the commit name when the commits are squashed. So your first comment of the PR, and the PR's title should follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/).

Reply:
Thanks @airblast-dev. +1 to everything there.

The inference part is described in each of the places where we've introduced `Into` in the breaking changes doc: Steal the wording from: https://github.com/ratatui/ratatui/blob/main/BREAKING-CHANGES.md#axislabels-now-accepts-intoiteratorintoline-1273-and-1283

Same deal with const - https://github.com/ratatui/ratatui/blob/main/BREAKING-CHANGES.md#block-style-methods-cannot-be-used-in-a-const-context-720 is an ok example.

Reply:
Put more concretely:
- Add a section to BREAKING-CHANGES.md that describes the inference issue. Copy it from somewhere else.
- Update the first PR comment to include a conventional commits `BREAKING CHANGES:` footer which has similar information. This will be used in the changelog when we publish Ratatui next.